src/main/java/spleetwaise/commons/MainApp.java

```diff
@@ -91,51 +90,38 @@ public void init() throws Exception {
         Runtime.getRuntime().addShutdownHook(new Thread(this::cleanUp));
     }
 
-    /**
-     * Returns a {@code AddressBookModelManager} with the data from {@code storage}'s address book and
-     * {@code userPrefs}. <br> The data from the sample address book will be used instead if {@code storage}'s address
-     * book is not found, or an empty address book will be used instead if errors occur when reading {@code storage}'s
-     * address book.
-     */
-    private AddressBookModel initAddressBookModelManager(Storage storage) {
-        logger.info("Using data file : " + storage.getAddressBookFilePath());
+    private void initModelManagers(Storage storage) {
+        logger.info("Using Ab file : " + storage.getAddressBookFilePath());
+        logger.info("Using Tb file : " + storage.getTransactionBookFilePath());
 
-        Optional<ReadOnlyAddressBook> addressBookOptional;
-        ReadOnlyAddressBook initialData;
         try {
-            addressBookOptional = storage.readAddressBook();
+            Optional<ReadOnlyAddressBook> addressBookOptional = storage.readAddressBook();
             if (!addressBookOptional.isPresent()) {
+                // If the addressbook is not present start from sample data
                 logger.info("Creating a new data file " + storage.getAddressBookFilePath()
                         + " populated with a sample AddressBook.");
-            }
-            initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
-        } catch (DataLoadingException e) {
-            logger.warning("Data file at " + storage.getAddressBookFilePath() + " could not be loaded."
-                    + " Will be starting with an empty AddressBook.");
-            initialData = new AddressBook();
-        }
-
-        return new AddressBookModelManager(initialData);
-    }
-
-    private TransactionBookModel initTransactionModelManager(Storage storage, AddressBookModel addressBookModel) {
-        logger.info("Using data file : " + storage.getTransactionBookFilePath());
-
-        Optional<ReadOnlyTransactionBook> txnBookOptional;
-        ReadOnlyTransactionBook initialData;
-        try {
-            txnBookOptional = storage.readTransactionBook(addressBookModel);
-            if (!txnBookOptional.isPresent()) {
                 logger.info("Creating a new data file " + storage.getTransactionBookFilePath()
                         + " populated with a sample TransactionBook.");
+                addressBookModel = new AddressBookModelManager(SampleDataUtil.getSampleAddressBook());
```

Comment:
Nit, should we move line 103 and line 104 to after line 105?
```suggestion
                addressBookModel = new AddressBookModelManager(SampleDataUtil.getSampleAddressBook());
                logger.info("Creating a new data file " + storage.getTransactionBookFilePath()
                        + " populated with a sample TransactionBook.");
```

Reply:
Hmm why though? the log message is to inform that there is no address book present